aten/src/ATen/core/interned_strings.h

```diff
@@ -56,7 +56,7 @@ namespace c10 {
   _(prim, ReturnStmt)                \
   _(prim, BreakStmt)                 \
   _(prim, ContinueStmt)              \
-  _(prim, ListComprehensionScope)    \
+  _(prim, ComprehensionScope)        \
```

Comment:
I think this breaks BC test, also might have ramification for mobile. Could we still keep `ListComprehensionScope`?

Reply:
`ListComprehension` isn't emitted as  an operator, it's a node place holder, just like prim::If, that exists temporarily in initial compilation. Changing the name doesn't effect BC (michael recently did it)